### PR TITLE
Fix condition preventing to run multivcfanalyzer

### PR DESCRIPTION
**Pull request by maxibor**
A bit of cleanup and fixing a parameter type to allow to run MultiVCFAnalyzer

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] `CHANGELOG.md` is updated.

### CODE REVIEW

**Comment by maxibor** [comment]
Merged dev branch back into PR branch to sync with template update @jfy133

File: main.nf [hunk 1]
```diff
@@ -2494,22 +2494,22 @@ process vcf2genome {
 // More complex consensus caller with additional filtering functionality (e.g. for heterozygous calls) to generate SNP tables and other things sometimes used in aDNA bacteria studies
 
 // Create input channel for MultiVCFAnalyzer, possibly mixing with pre-made VCFs.
-if (params.additional_vcf_files == '') {
-    ch_vcfs_for_multivcfanalyzer = ch_ug_for_multivcfanalyzer.map{ it[7] }.collect()
+if (!params.additional_vcf_files) {
+    ch_vcfs_for_multivcfanalyzer = ch_ug_for_multivcfanalyzer.map{ it[-1] }.collect()
```
Comment (jfy133) [inline main.nf:2498]: Any reason why you recommend `-1` vs `7`? I went for the latter to enforce very strict structure of the TSV and subsequent tuples?

**Review by jfy133 — CHANGES_REQUESTED** [review]
LGTM, just need a CHANGELOG update and good to go

File: main.nf (hunk 1 above)
Comment (maxibor) [reply main.nf:2498]: Because I was too lazy to keep track of how many there were :joy:

**Comment by maxibor** [comment]
Changelog updated @jfy133

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Fix markdown lint and you can merge @maxibor! (ignore nf-core linting issue)

**Review by jfy133 — APPROVED** [review]

**Pull request merged**